TPP.Core.Tests/Commands/Definitions/BadgeCommandsTest.cs

```diff
@@ -306,5 +306,27 @@ public async Task TestGiftBadgeNotOwned()
                     It.IsAny<IDictionary<string, object?>>()),
                 Times.Never);
         }
+
+        [Test]
+        public async Task ListSellBadge_lists_callers_badges_when_user_isnt_specified()
+        {
+            User user1 = MockUser("u1");
+            PkmnSpecies speciesA = PkmnSpecies.RegisterName("1", "a");
+            Badge badgeA = new("badgeA", user1.Id, speciesA, Badge.BadgeSource.Pinball, Instant.MinValue, 0, false) {SellPrice = 1 };
+
+            _badgeRepoMock.Setup(repo => repo.FindAllForSaleByCustom(user1.Id, speciesA, null, null, false)).Returns(Task.FromResult(new List<Badge>() { badgeA }));
+            _userRepoMock.Setup(repo => repo.FindById(user1.Id)).Returns(Task.FromResult((User?)user1));
+
+            _argsParser.AddArgumentParser(new PkmnSpeciesParser(new[] { speciesA }));
+            _argsParser.AddArgumentParser(new ShinyParser());
+            _argsParser.AddArgumentParser(new BadgeSourceParser());
+            _argsParser.AddArgumentParser(new StringParser());
+
+            await _badgeRepoMock.Object.SetBadgeSellPrice(badgeA, 1);
```

Comment:
this line probably has no effect. You're calling a method on a mock. The return value of `FindAllForSaleByCustom` is already mocked above